src/parser/classes/Factoid.ts

```diff
@@ -0,0 +1,16 @@
+import { YTNode } from '../helpers.js';
+import { type RawNode } from '../index.js';
+
+export default class Factoid extends YTNode {
+  static type = 'Factoid';
+  label: String;
+  value: String;
+  accessibility_text: String;
+
+  constructor(data: RawNode) {
+    super();
+    this.label = data.label.simpleText;
```

Comment:
Again, using the misc class `Text` here would be more safe. And to keep the type as a `string`, you can call `Text#toString` after instantiating it.